RoddyCore/src/de/dkfz/roddy/config/ToolEntry.java

```diff
@@ -217,6 +219,39 @@ public ToolStringParameter clone() {
         }
     }
 
+    public static class ToolFileParameterCondition {
+
+        private String condition;
+
+        private Boolean pureBoolean;
+
+        public ToolFileParameterCondition(boolean pureBoolean) {
+            this.pureBoolean = Boolean.valueOf(pureBoolean);
+        }
+
+        public ToolFileParameterCondition(String condition) {
+            if(condition.startsWith("conditional:"))
+                this.condition = condition.substring(12);
+            else {
+                this.pureBoolean = RoddyConversionHelperMethods.toBoolean(condition, true);
```

Comment:
Actually this will not be used at all. But: The default value for check IS true. So I think in this case it is ok. When the value comes in via a configuration value, the normal checks are done upon config read-in.